code/part-two/README.md

```diff
@@ -135,10 +134,10 @@ docker exec -it shell bash
 
 An important note! When you are within a container, you cannot use "localhost"
 for networking. Remember how each container is like its own little computer?
-Well it is complete with it's own localhost. Thankfully, Docker provide the
-service names from the compose file as URLs. So for example, if you were inside
-the shell container and wanted to get the blocks from the REST API, you would
-run:
+Well it comes complete with it's own localhost. For networking to _other_
```

Comment:
**Add comma:** Well`,` it comes complete...
**Homophone:** ...it comes complete with ~it's~ `its` own localhost